test/TestLogger.AcceptanceTests/Snapshots/TestLoggerAcceptanceTests/VerifyTestRunOutput/Json.TestLogger.NUnit.NetCore.Tests-;Parser=Legacy-IncludesParserFailures.verified.txt

```diff
@@ -171,90 +131,54 @@
             Method: Json.TestLogger.NUnit.NetCore.Tests.NetCoreOnly.Issue28_Examples.ExampleTest2(True,4.1m,5.0m,(5.0, True)),
             Result: Passed
           },
           {
-            FullyQualifiedName: Json.TestLogger.NUnit.NetCore.Tests.NetCoreOnly.Issue28_Examples.ExampleTest2(True,4.8m,4.5m,(4.8, False)),
-            DisplayName: ExampleTest2(True,4.8m,4.5m,(4.8, False)),
             Namespace: UnknownNamespace,
             Type: UnknownType,
             Method: Json.TestLogger.NUnit.NetCore.Tests.NetCoreOnly.Issue28_Examples.ExampleTest2(True,4.8m,4.5m,(4.8, False)),
             Result: Passed
           },
           {
-            FullyQualifiedName: Json.TestLogger.NUnit.NetCore.Tests.NetCoreOnly.Issue28_Examples.ExampleTest2(True,4.5m,4.5m,(4.5, False)),
-            DisplayName: ExampleTest2(True,4.5m,4.5m,(4.5, False)),
             Namespace: UnknownNamespace,
             Type: UnknownType,
             Method: Json.TestLogger.NUnit.NetCore.Tests.NetCoreOnly.Issue28_Examples.ExampleTest2(True,4.5m,4.5m,(4.5, False)),
             Result: Passed
           },
           {
-            FullyQualifiedName:
-NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples.ExampleTest4(System.AggregateException: One or more errors occurred. (Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.)
- ---> NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1: Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.
-   --- End of inner exception stack trace ---),
-            DisplayName:
-ExampleTest4(System.AggregateException: One or more errors occurred. (Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.)
- ---> NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1: Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.
-   --- End of inner exception stack trace ---),
             Namespace: UnknownNamespace,
             Type: UnknownType,
-            Method:
-NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples.ExampleTest4(System.AggregateException: One or more errors occurred. (Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.)
- ---> NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1: Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.
-   --- End of inner exception stack trace ---),
+            Method: NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples.ExampleTest4(System.AggregateException: One or more errors occurred. (Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.)\n ---> NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1: Exception of type 'NUnit.Xml.TestLogger.NetFull.Tests.Issue28_Examples+MyException1' was thrown.\n   --- End of inner exception stack trace ---),
```

Comment:
Looks like this may be an actual change (also below). We must have been removing the line breaks in the name before. 

It may be that the json sanitizer is running before some other code that would strip out the, now escaped, line breaks.

Reply:
- [ ] Need to revisit `x = x.Replace(@"\r\n", @"\n"); // Fix cross plat failures.` depending on what I find here.

Reply:
I looked at it some more and don't think this is an issue. Just the result of the json sanitizer removing line breaks.